device/bgppreferencesubtlv.yaml

```diff
@@ -0,0 +1,17 @@
+components:
+  schemas:
+    Device.BgpPreferenceSubTlv:
+      description: >-
+        Container for BGP preference sub TLV settings.
+      type: object
+      properties:
+        enable:
```

Comment:
Since we now support assymetric configuration natively in the revised model , the enable field might not be needed. The presence of this control was needed mostly to create assymetric configurations in a symmetric model . Now the subtlv can be selectively added only where required.